plugin/pkg/admission/noderestriction/admission_test.go

```diff
@@ -149,12 +149,11 @@ func setAllowedUpdateLabels(node *api.Node, value string) *api.Node {
 	node.Labels["kubernetes.io/hostname"] = value
 	node.Labels["failure-domain.beta.kubernetes.io/zone"] = value
 	node.Labels["failure-domain.beta.kubernetes.io/region"] = value
+	node.Labels["topology.kubernetes.io/zone"] = value
+	node.Labels["topology.kubernetes.io/region"] = value
 	node.Labels["beta.kubernetes.io/instance-type"] = value
 	node.Labels["beta.kubernetes.io/os"] = value
 	node.Labels["beta.kubernetes.io/arch"] = value
-	node.Labels["failure-domain.kubernetes.io/zone"] = value
-	node.Labels["failure-domain.kubernetes.io/region"] = value
-	node.Labels["kubernetes.io/instance-type"] = value
```

Comment:
restore this line

Reply:
That's for `failure-domain.kubernetes.io/zone` not `failure-domain.beta.kubernetes.io/zone`. Still restore it?

Reply:
oh, you're referring to instance type, nvm